.changeset/polite-eels-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Added accessible button name to expand button.
```

Comment:
which expand button?

Reply:
Just changed it to `Added accessible button name to SideNavigation expand button.`, hopefully that's enough context?